test/tests/ver-1a/tests

```diff
@@ -1,11 +1,18 @@
 [Tests]
   design = 'EnclosureSinkScalarKernel.md PressureReleaseFluxIntegral.md EquilibriumBC.md'
-  issues = '#12'
-  [exo]
+  issues = '#12 #75'
+  [ver-1a]
     type = Exodiff
     input = ver-1a.i
-    exodiff = ver-1a_out.e
-    cli_args = 'Executioner/dt=10 Executioner/scheme=implicit-euler'
+    exodiff = ver-1a_test_out.e
+    cli_args = 'Executioner/dt=5 Mesh/nx=10 Outputs/file_base=ver-1a_test_out'
     requirement = 'The system shall be able to model species, originating from a depleting source enclosure, diffusion through a structure.'
   []
+  [ver-1a_heavy]
+    type = Exodiff
+    input = ver-1a.i
+    exodiff = ver-1a_out.e
+    requirement = 'The system shall be able to model species, originating from a depleting source enclosure, diffusion through a structure, with the fine mesh and timestep required to match the analytical solution.'
+    valgrind = 'HEAVY'
```

Comment:
```suggestion
    heavy = true
```

@simopier Noticed that CI wasn't using `--all` and wasn't designating this test as heavy, so I looked into it. This is the proper way to designate "heavy" for the Test Harness. The way you designated was only for valgrind CI runs, which we don't currently do for TMAP8.

Reply:
Thanks for looking into it!